accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -2,6 +2,13 @@
 Chile
 =====
 
+Webinars
+========
+Below you can find videos with a general description of the localization, how to configure it and 
+what changes were made for version 14.
+
+- `VIDEO WEBINAR OF A COMPLETE DEMO <https://youtu.be/BHnByZiyYcM>`_.
+
 Introduction
 ============
```

Comment:
May I suggest to put it as a note, at the very beginning of the intro, rather than making a whole section just for one link?

It would look like this:
![image](https://user-images.githubusercontent.com/46810843/104744864-aeb30c00-574d-11eb-8b3f-af4bf1a578ee.png)


```suggestion
Introduction
============

.. note::
   Here is a `webinar video on YouTube with a general description of the localization and how to
   configure it <https://youtu.be/BHnByZiyYcM>`_.
```